announcer.js

```diff
@@ -7,47 +7,66 @@ const base = new Airtable({
     apiKey: process.env.AIRTABLE_API_KEY
 }).base(process.env.AIRTABLE_BASE);
 
-base('Slack Announcer').select({
-    view: "Announcer Filter"
-}).eachPage(function page(records, fetchNextPage) {
+var airtableCronJobs = [];
 
-    // This function (`page`) will get called for each page of records.
+// Flush and reload the CRON table at 3 minutes and 3 seconds past every hour
+// This is specifically offset from 5, 10, 15 minute intervals to ensure that 
```

Comment:
Do you think it would be good to raise a warning about this when loading the cron jobs?